src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -101,7 +101,7 @@ public KubernetesAppDeployer(KubernetesDeployerProperties properties, Kubernetes
     public String deploy(AppDeploymentRequest request) {
         String appId = createDeploymentId(request);
         if (logger.isDebugEnabled()) {
-            logger.debug(String.format("Deploying app: %s", appId));
+            logger.debug(String.format("Deploying app: %s, request: %s", appId, request));
```

Comment:
Is there any sensitive info in the request object?

Reply:
Updated by copying ArgumentSanitizer from dataflow project.